.github/scripts/setup/base-dependencies.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+
+sudo apt -y update
+sudo apt install -y --no-install-recommends \
+	build-essential \
+	sudo \
```

Comment:
This script already uses `sudo`, means sudo package is present no need installation